docs/interpreter/jdbc.md

```diff
@@ -827,6 +827,38 @@ Dependencies
   </tr>
 </table>
 
+### Apache Kyuubi
+
+Properties
+
+<table class="table-configuration">
+  <tr>
+    <th>Name</th>
+    <th>Value</th>
+  </tr>
+  <tr>
+    <td>default.driver</td>
+    <td>org.apache.hive.jdbc.HiveDriver</td>
+  </tr>
+  <tr>
+    <td>default.url</td>
+    <td>jdbc:hive2://kyuubi-server:2181/default;serviceDiscoveryMode=zooKeeper;zooKeeperNamespace=kyuubi</td>
+  </tr>
+</table>
+
+Dependencies
+
+<table class="table-configuration">
+  <tr>
+    <th>Artifact</th>
+    <th>Excludes</th>
+  </tr>
+  <tr>
+    <td>org.apache.hive:hive-jdbc:2.3.4</td>
```

Comment:
cc @pan3793

Reply:
We recommend using `org.apache.kyuubi:kyuubi-hive-jdbc-shaded:1.5.2-incubating` to connect Kyuubi.

https://kyuubi.readthedocs.io/en/latest/client/kyuubi_jdbc.html